tests/test_multidict.py

```diff
@@ -387,6 +388,23 @@ def test_and2(self, cls: Type[MutableMultiMapping[str]]) -> None:
 
         assert {"key"} == {"key", "key2"} & d.keys()
 
+    def test_and_not_implemented(self, cls: Type[MutableMultiMapping[str]]) -> None:
+        d = cls([("key", "value1")])
+
+        sentinel_operation_result = object()
+
+        class RightOperand:
+            def __rand__(self, other):
```

Comment:
@a5r0n so MyPy demands annotations here: https://results.pre-commit.ci/run/github/51206425/1706753865.XjT9kVUPShKRS-PfstvZaQ.

Reply:
Done

Reply:
Looks like it wants type args there: https://results.pre-commit.ci/run/github/51206425/1706754519.rj18BkaQT4qOcCG8QRnS0A.

Reply:
I see, thats should be `KeysView[str]` i guess?

Reply:
Seems so.

Reply:
Marking as resolved.